src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -61,104 +61,133 @@ static ColliderCylinderInit sCylinderInit = {
     { 30, 40, 0, { 0, 0, 0 } },
 };
 
-AnimationHeader* D_80B6D1C8[] = {
-    &ovl_En_Sth_Anim_003F50, &ovl_En_Sth_Anim_0045B4, &ovl_En_Sth_Anim_004CC0, &ovl_En_Sth_Anim_00533C,
-    &ovl_En_Sth_Anim_0059B8, &ovl_En_Sth_Anim_005E40, &ovl_En_Sth_Anim_006354, &ovl_En_Sth_Anim_00645C,
+typedef enum {
+    /* 0x0 */ STH_ANIM_SIGNALLING,   // default, waving arms at you from telescope, OOT: cured happy animation
+    /* 0x1 */ STH_ANIM_BENDING_DOWN, // default anim of cured spider house, but never seen before wait overrides it
+    /* 0x2 */ STH_ANIM_TALK,
+    /* 0x3 */ STH_ANIM_WAIT,
+    /* 0x4 */ STH_ANIM_LOOK_UP,     // South Clock Town, looking at moon
+    /* 0x5 */ STH_ANIM_LOOK_AROUND, // checking out Oceanside Spider House
+    /* 0x6 */ STH_ANIM_PLEAD,       // wants to buy Oceanside Spider House
+    /* 0x7 */ STH_ANIM_PANIC,       // after buying Oceanside Spider House, can be found at bottom of slide,
+    /* 0x8 */ STH_ANIM_START,       // set in init, not an actual index to the array
+} EnSthAnimationIndexes;
```

Comment:
```suggestion
} EnSthAnimation;
```
This enum usually just has `Animation`. Can also probably be in dec as well